src/testjfxapp/MainMenu.java

```diff
@@ -193,7 +193,7 @@ public void start(Stage primaryStage) throws Exception {
         window.setTitle("Tetsaw Main Menu");
         window.setMinWidth(300);
         window.setMinHeight(500);
-        window.getIcons().add(new Image("/icon.png"));
+//        window.getIcons().add(new Image(getClass().getClassLoader().getResource("/icon.png").toExternalForm()));
```

Comment:
Might want to make sure this debug attempt isn't left in, though we can revert it back to normal at some point.